src/lavatory/commands/stats.py

```diff
@@ -13,12 +13,31 @@
 @click.option(
     '--repo',
     default=None,
+    multiple=True,
     required=False,
-    help='Name of specific repository to run against. If not provided, uses all repos.')
+    help='Name of specific repository to run against. Can use --repo multiple times. If not provided, uses all repos.')
 def stats(ctx, repo):
-    """Get statistics of a repo."""
+    """Get statistics of repos."""
     LOG.debug('Passed args: %s, %s.', ctx, repo)
     artifactory = Artifactory(repo_name=repo)
```

Comment:
What happens here if you pass a tuple/list in as `repo_name`?

Reply:
I have a logic below this line.

Reply:
yeah, but `Artifactory(repo_name=repo)` is going to be passing in `repo=("repo1", "repo2", etc)`. Does that work?

Reply:
That is not part of my change. But yes you can pass anything it will accept that. Stats do work. But in Artifactory class we need to refactor according to multiple stuff.

Reply:
I have initialized Artifactory with None.